src/subPages/SimilarProteins/Table/index.tsx

```diff
@@ -0,0 +1,262 @@
+    '@type': 'PropertyValue',
+    name: 'SimilarProteins',
+    value: data.map(
+      (p) =>
+        `https://www.ebi.ac.uk/interpro/protein/${p.metadata.source_database}/${p.metadata.accession}/`,
+    ),
+  };
+};
+
+const getAPIURLForSimilarProteins = (
+  { protocol, hostname, port, root }: ParsedURLServer,
+  ida: string,
+  db: string,
+) =>
+  format({
+    protocol,
+    hostname,
+    port,
+    pathname:
+      root +
+      descriptionToPath({
+        main: { key: 'protein' },
+        protein: { db: db },
+      }),
+    query: { ida },
+  });
+
+type Props = {
+  db: string;
+  ida: string;
+  search?: Record<string, string>;
+  description?: InterProDescription;
+  api?: ParsedURLServer;
+};
+interface LoadedProps
+  extends Props,
+    LoadDataProps<PayloadList<ProteinMetadata>, 'IDA'> {}
+
+const SimilarProteinTable = ({
+  dataIDA,
+  isStaleIDA,
+  db,
+  ida,
+  description,
+  search,
+  api,
+}: LoadedProps) => {
+  const { loading, payload, url } = dataIDA || {};
+  if (loading || !payload) return <Loading />;
+  if (!description || !api) return null;
+  return (
+    <>
+      <SchemaOrgData data={payload.results} processData={schemaProcessData} />
+      <Table
+        dataTable={payload.results}
+        actualSize={payload.count}
+        query={search}
+        isStale={isStaleIDA}
+        currentAPICall={url}
+        nextAPICall={payload.next}
+        previousAPICall={payload.previous}
+        notFound={payload.results.length === 0}
+      >
+        <PageSizeSelector />
+        <Exporter>
+          <div className={css('menu-grid')}>
+            <label htmlFor="fasta">FASTA</label>
+            <AllProteinDownload
+              description={description}
+              ida={ida}
+              db={db}
+              count={payload.count}
+              fileType="fasta"
+            />
+            <label htmlFor="json">JSON</label>
+            <AllProteinDownload
+              description={description}
+              ida={ida}
+              db={db}
+              count={payload.count}
+              fileType="json"
+            />
+            <label htmlFor="tsv">TSV</label>
+            <AllProteinDownload
+              description={description}
+              ida={ida}
+              db={db}
+              count={payload.count}
+              fileType="tsv"
+            />
+            <label htmlFor="api">API</label>
+            <Link
+              target="_blank"
+              href={getAPIURLForSimilarProteins(api, ida, db)}
+              className={css(
+                'vf-button',
+                'vf-button--secondary',
+                'vf-button--sm',
+                'generate-button',
+              )}
+            >
+              <span
+                className={css('icon', 'icon-common', 'icon-export')}
+                data-icon="&#xf233;"
+              />{' '}
+              <span className={css('file-label')}>Web View</span>
+            </Link>
```

Comment:
Has a slightly different look that other buttons. Could be make exactly like other buttons, if possible.